Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.15.0
+
+USER root
+
+RUN apk upgrade --no-cache && apk add --no-cache openjdk17-jre
+
+ENV USER user_hocs
+ENV USER_ID 10000
```

Comment:
Is this the right USER ID? We use various ones - I believe 1000+ is fine for what we need but worth a check.

Reply:
1000+ are likely to exist on the host system, meaning if you can escape the docker sandbox you have permissions on the host too, we ought to change this to chown 10000:10001

https://github.com/hexops/dockerfile#do-not-use-a-uid-below-10000